stable/ghost/templates/deployment.yaml

```diff
@@ -17,7 +17,8 @@ spec:
     spec:
       initContainers:
       - name: volume-permissions
-        image: busybox
+        image: "{{ .Values.initContainerImage }}"
+        imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
```

Comment:
@sameersbn Thanks for the clarity, my concern is only about the new `initContainers` that's being introduced as a part of this PR not the existing container.  My other concern was if we do not get `initConainer` structured this time (when it is introduced) we may find it hard to bring it later as that mightbecome a breaking change then.  What do you think?